src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -133,20 +101,14 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
 
 
         [Test]
-        public void Setter_dependencies_should_be_supported()
+        public void Setter_injection_should_be_enabled_by_default()
         {
             using (var builder = TestContainerBuilder.ConstructBuilder())
             {
                 builder.Configure(typeof(SomeClass), DependencyLifecycle.InstancePerCall);
                 builder.Configure(typeof(ClassWithSetterDependencies), DependencyLifecycle.SingleInstance);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "EnumDependency", SomeEnum.X);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "SimpleDependency", 1);
-                builder.ConfigureProperty(typeof(ClassWithSetterDependencies), "StringDependency", "Test");
-
-                var component = (ClassWithSetterDependencies) builder.Build(typeof(ClassWithSetterDependencies));
-                Assert.AreEqual(component.EnumDependency, SomeEnum.X);
-                Assert.AreEqual(component.SimpleDependency, 1);
-                Assert.AreEqual(component.StringDependency, "Test");
+
+                var component = (ClassWithSetterDependencies)builder.Build(typeof(ClassWithSetterDependencies));
```

Comment:
That API isn't available on this level, it's only present at the [`IBuilder`](https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/ObjectBuilder/IBuilder.cs) level.

these tests operate at the [`IContainer`](https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/ObjectBuilder/Common/IContainer.cs) level. Ok to leave as is?